pyproject.toml

```diff
@@ -0,0 +1,4 @@
+[tool.black]
+line-length = 88
+target_version = ["py36", "py37", "py38"]
```

Comment:
Maybe we can have here also 3.9 an 3.10 since they are supported (see `setup.py`)